assets/global.js

```diff
@@ -1315,39 +1085,24 @@ class ProductRecommendations extends HTMLElement {
   }
 
   connectedCallback() {
```

Comment:
Recommendation reinitialization now happens declaratively in the ProductInfo class